### ISSUE TITLE
Can we able to connect other SQL and NoSQL databases

### ISSUE THREAD

**Issue by sainisanjay** [dataprofessor/llama2#5]
I would like to know, Can we able to connect other SQL and NoSQL databases like mongodb databases to make a natural language query on its data. Is that possible?!

**Comment by w453y** [comment]
Yes, it is possible to connect to both SQL and NoSQL databases, including MongoDB, to perform natural language queries on their data. This typically involves implementing a system that supports database connectivity and integrating natural language processing (NLP) capabilities. The feasibility depends on the specific technologies and tools you choose for your project. If you have more details or specific requirements, I can provide more targeted information.
You can checkout this for database integration [Github_repo](https://github.com/w453y/llama2-chatbot-db-integration)
